### DISCUSSION TITLE
加载的模型，设置位置偏移和旋转角度没有成功

### DISCUSSION THREAD

**Discussion by DelingAlieZ10** [zouyaoji/vue-cesium#560] [Q&A] [answered]
设置 TranslationRotationScale 节点转换参数。结构：{translation: {x: number, y: number, z: number}, rotation: {x: number, y: number, z: number, w: number}, scale:{x: number, y: number, z: number}} 设置了模型在地图上的显示没有变化

**Comment by DelingAlieZ10** [comment]
<vc-entity :position="[110, 40, 1.0]" description="Hello VueCesium" @click="onEntityEvt" @mouseover="onEntityEvt" @mouseout="onEntityEvt">
                <vc-graphics-model ref="model" uri="/models/factory.glb" :scale="0.5" :nodeTransformations="nodeTransformations"></vc-graphics-model>
            </vc-entity>

导入了一个模型，想适当旋转角度怎么旋转啊

**Reply by zouyaoji** [reply, marked as answer]
旋转参数是entity的orientation参数。

**Reply by zouyaoji** [reply]
TranslationRotationScale 需要开启shouldAnimate 应该才可以

 参考一下  https://zouyaoji.top/vue-cesium/#/zh-CN/component/overlays/vc-overlay-dynamic
